### PR TITLE
feat: apply project level permissions + e2e tests

### PR DESCRIPTION
**Pull request by rephus**
Closes: [#2437](https://github.com/lightdash/lightdash/issues/2437)

### Description:
Peek 2022-06-22 10-19
Peek 2022-06-22 10-22

### CODE REVIEW

File: packages/e2e/cypress/e2e/api/projectPermissions.cy.ts [hunk 1]
```diff
@@ -0,0 +1,801 @@
...
+    sorts: [{ fieldId: 'customers_customer_id', descending: false }],
+    limit: 500,
+    tableCalculations: [],
+    additionalMetrics: [],
+};
+const sqlQueryBody = { sql: 'select 1' };
+describe('Lightdash API tests for member user with admin project permissions', () => {
+    let email;
+    beforeEach(() => {
+        cy.loginWithPermissions('member', [
```
Comment (rephus) [inline packages/e2e/cypress/e2e/api/projectPermissions.cy.ts:18]: big test file where we check all endpoints with different project permissions for `member` users

File: packages/e2e/cypress/support/commands.ts [hunk 2]
```diff
@@ -77,9 +84,93 @@ Cypress.Commands.add('registerNewUser', () => {
...
         method: 'GET',
     });
 });
+
+type ProjectPermission = {
+    role: string;
+    projectUuid: string;
+};
+Cypress.Commands.add(
+    'loginWithPermissions',
```
Comment (rephus) [inline packages/e2e/cypress/support/commands.ts:156]: New command to : 
- invite user with X role 
- add X role to project 
- register with invitation code

File: packages/e2e/cypress/e2e/projectPermission.cy.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import { SEED_PROJECT } from '@lightdash/common';
+
+describe('Login', () => {
+    it('Organization admin can see projects', () => {
+        cy.login();
+
+        cy.visit('/');
+        cy.contains('Settings');
+        cy.get('.bp4-non-ideal-state').should('not.exist');
```
Comment (rephus) [inline packages/e2e/cypress/e2e/projectPermission.cy.ts:9]: New projectPermission e2e test to check the UI

**Review by ZeRego — APPROVED** [review]
*chef kiss* 🎉

**Pull request merged**